swap_meet/clothing.py

```diff
@@ -1,2 +1,8 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Clothing(Item):
+    def __init__(self, category="Clothing", condition=0) -> None:
+        super().__init__(category, condition)
```

Comment:
This setup makes it possible to pass in a value for category that is not "Clothing", which could result in an instance of `Clothing` that has some other category.  
I notice that you're using [type hints](https://typing.readthedocs.io/en/latest/#) here (` -> None`)for the constructor.  In python, type hints are not used for type checking, but it is possible to add type checkers or linters that use the type hints.  From a stylistic standpoint, I would not recommend using type hints for just one function, or one part of one function.  It's fine to use them, but I'd recommend using them in a more comprehensive way or not at all.
```suggestion
    def __init__(self, condition=0):
        super().__init__("Clothing", condition)
```